doc/api/buffer.md

```diff
@@ -1687,9 +1687,9 @@ console.log(buf.readInt32LE(1));
 added: v0.11.15
 -->
 
-* `offset` {integer} Where to start reading. Must satisfy: `0 <= offset <= buf.length - byteLength`
-* `byteLength` {integer} How many bytes to read. Must satisfy: `0 < byteLength <= 6`
-* `noAssert` {boolean} Skip `offset` and `byteLength` validation? **Default:** `false`
+* `offset` {integer} Where to start reading. Must satisfy: `0 <= offset <= buf.length - byteLength`.
```

Comment:
I'm not sure if it should be addressed in this PR, but there are lines longer than 80 characters.